layers/meta-balena-jetson/recipes-kernel/linux/linux-tegra_%.bbappend

```diff
@@ -219,6 +229,7 @@ BALENA_CONFIGS_DEPS[cdc-ncm] = " \
 "
 
 BALENA_CONFIGS_append_photon-nano = " mii"
+BALENA_CONFIGS_append_photon-tx2-nx = " mii"
```

Comment:
same for this and the other configs above?

Reply:
Ethernet is working